internal/tools/policyenginesimulator/samples/simple_framework/commands.json

```diff
@@ -5,7 +5,7 @@
             "feed": "contoso.azurecr.io/infra",
             "issuer": "did:web:contoso.com",
             "namespace": "fragment",
-            "local_path": "relative/path/to/local/fragment.rego"
```

Comment:
Does this really need to change? Is this not supposed to be a relative path?

Reply:
@anmaxvl It needs to change, because it fails when you try to run it. The custom policy one does work as it points to the right path.

I'd expect the example in here to work. Custom policy works as it points to the correct file (https://github.com/microsoft/hcsshim/blob/main/internal/tools/policyenginesimulator/samples/simple_custom/commands.json#L8) 

Am I missing any reasoning to keep it this way even though it fails when you run it? This wrong path seems to be serving no purpose?

Reply:
I was treating this as an example and not necessarily an actual path that should be used, but I don't feel strongly about it either.